ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/photo/PhotoTaskFragment.kt

```diff
@@ -55,11 +54,7 @@ class PhotoTaskFragment : Hilt_PhotoTaskFragment<PhotoTaskViewModel>() {
   private var capturedPhotoPath: String? = null
 
   override fun onCreateTaskView(inflater: LayoutInflater, container: ViewGroup?): TaskView =
-    TaskViewFactory.createWithoutHeader(
-      inflater,
-      R.drawable.outline_photo_camera,
-      R.string.take_a_photo
-    )
```

Comment:
We still need to show the icon and label somewhere. The lint will complain to remove these resources in the same CL if we don't refer to it anywhere.

Reply:
I removed the label altogether because we have another string for "Take a picture", the icon I added to the "Take a picture" button.

Reply:
I installed the app at HEAD and it looks like the header is already visible in drop a pin and draw polygon tasks.

Reply:
Yea I think the consensus for those screens was to always show the header, and then add the info button that shows the additional hint text if necessary.